workfinder/search/s1_ard.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import os
+from pathlib import Path
+
+import pandas as pd
+
+from workfinder import get_config
+from workfinder.search import list_s3_files, nats_connect, nats_publish, nats_close, list_catalog, get_ard_list
+from workfinder.search.BaseWorkFinder import BaseWorkFinder
+
+
+class S1ARD (BaseWorkFinder):
+
+    def submit_tasks(self, to_do_list):
```

Comment:
Some unit tests for these implementations would be great, doing this is a very good exercise to understand how coupled your function is to things like, for instance, nats, which in this case I would personally pass it as an argument rather than doing the connection inside